### ISSUE TITLE
Convert all upgrades to be shared. = false works incorrect

### ISSUE THREAD

**Issue by dkathlee** [Malcolm-Q/LC-LateGameUpgrades#517]
I set "Convert all upgrades to be shared" to false to upgrade better scanner separately and set all other upgrades "Individual Purchase" to false.
But after loading save better scanner forces to maximum upgraded level (i.e. one player upgrades it to 3 level while others only to 1 level and after loading into save all players have level 3).
Another issue with night vision. In described config glasses applyed to host when any non host player applyes glasses and host losing his glasses when any non host player with glasses dies.

**Comment by WhiteSpike** [comment]
The leveling issue only happens with better scanner? The night vision dropping for everyone even when individual is known and I still haven't found the cause of it.

I can only look into this after my break (which should end on 29th)

**Comment by dkathlee** [comment]
Problem with leveling also was on Night Vision batteries. I can`t describe it (cos it we didn`t understand logic of this bug), but on save load NV Battery upgrades was not on saved level (and we should upgrade it separately despite "Individual Purchase" was set to false). We resolved this problem by upgrading all players NV Battery to max level. Problems with leveling and NV glasses appears only after setting "Convert all upgrades to be shared" to false and set "Individual Purchase" to false (exept better scaner, it was set to true). With "Convert all upgrades to be shared" to true everything was OK

**Comment by WhiteSpike** [comment]
I believe I fixed the issue with setting individual not grabbing the correct saves.

The night vision being disabled on all people whenever a client dies is the only thing I cannot figure out what's going on right now.

**Comment by WhiteSpike** [comment]
Closing due to inactivity.
